tools/manifest-templator/manifest-templator.go

```diff
@@ -415,17 +466,20 @@ func main() {
 		CNA:      &operatorData{OperatorTag: *networkAddonsTag, ComponentTag: *networkAddonsTag},
 		SSP:      &operatorData{OperatorTag: *sspTag, ComponentTag: *sspTag},
 		NMO:      &operatorData{OperatorTag: *nmoTag, ComponentTag: *nmoTag},
+		MRO:      &operatorData{OperatorTag: *mroTag},
 	}
 	data.CreatedAt = time.Now().String()
 
 	// Load in all HCO Resources
-	getHCO(&data)
+	getHCO(data)
```

Comment:
My understanding is that this becomes `getHCO(&data)` since the function accepts a pointer to `templateData` but I would prefer to be explicit unless there is some meaningful or idiomatic reason for it being written this way.

Reply:
I changed
```
data := &templateData{
...
}
to be pointed instead of each time to use `&data`

Reply:
Missed it on the first read. Thanks.